pages/_app.js

```diff
@@ -67,18 +68,22 @@ export default class HomePage extends App {
               </Col>
               <Col medium={8} small={4} xsmall={2} large={8}>
                 <nav
+                  aria-label="Navigation Menu"
                   id="visibleDesktopMenu"
                   className="VisibleMenu  FlexEndPositioning"
                 >
-                  <ul className="NavList">
-                    <li className="DisplayNoneOnSmallScreen">
+                  <ul role="menu" className="NavList">
+                    <li role="menuitem" className="DisplayNoneOnSmallScreen">
                       <a href="/cathocomponents">Catho Components</a>
                     </li>
-                    <li className="DisplayNoneOnSmallScreen">
+                    <li role="menuitem" className="DisplayNoneOnSmallScreen">
                       <a href="/quantum">Quantum</a>
                     </li>
                     <li>
                       <button
+                        aria-controls="hiddenDesktopMenu"
+                        aria-expanded="false"
```

Comment:
This is a dynamic property that must be "true" when the menu is open. Can you do this using useState? The idea is that when clicking the button, a Boolean variable in the state would be changed and used to define the value of aria-expanded. In the catho-components header there is a basis on how you can do this

Reply:
I made a mistake here when making this suggestion. I saw that the component is a class and not a function and therefore it is not possible to use useState to implement the aria-expanded dynamics. However, I see the opportunity to implement the dynamics through pure JavaScript, within the listener that handles the menu button click. I will open a technical debt for this